spec/models/listing_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Listing, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
instead of pending, replace this with: 
```ruby
describe 'associations' do
  subject(build(:listing))

  it { is_expected.to belong_to(:user) }
end
```